modules/nw-osp-services-external-load-balancer.adoc

```diff
@@ -33,6 +39,14 @@ Red Hat supports the following services for a user-managed load balancer:
 * OpenShift API
 * OpenShift MachineConfig API
 
+
+ifdef::post-bare-metal[]
```

Comment:
I would still prefer to just remove this from the post-install docs. We expect the new user-managed loadbalancer mechanism to replace the old post-install one. If it turns out a lot of customers still want to use the old way for some reason we can always put it back.

Reply:
Thanks, removed.